lms/djangoapps/grades/tests/test_transformer.py

```diff
@@ -330,6 +353,38 @@ def test_course_version_collected_in_split(self):
         block_structure = get_course_blocks(self.student, blocks[u'course'].location, self.transformers)
         self.assertIsNotNone(block_structure.get_xblock_field(blocks[u'course'].location, u'course_version'))
 
+    def test_grading_policy_collected(self):
+        # the calculated hash of the original and updated grading policies of the test course
+        original_grading_policy_hash = u'ChVp0lHGQGCevD0t4njna/C44zQ='
+        updated_grading_policy_hash = u'TsbX04qWOy1WRnC0NHy+94upPd4='
+
+        blocks = self.build_course_with_problems()
+        course_in_modulestore = modulestore().get_course(blocks[u'course'].location.course_key)
```

Comment:
Rather than getting the course back from the modulestore, you can just do: `blocks['course']`.